taskfiles/hosts.yml

```diff
@@ -13,12 +13,20 @@ tasks:
       - gum confirm "Simulated changes above. Are you sure you want to continue?"
       - ansible-playbook -vv launchpad-core/ansible/site.yaml -i {{.INVENTORY_PATH}} {{.CLI_ARGS}}
 
+  apply-zfs:
+    interactive: true
+    desc: Performs base host configuration, including package installations and configuring SSH
+    cmds:
+      - ansible-playbook -vv launchpad-core/ansible/zfs_install.yaml --check --diff -i {{.INVENTORY_PATH}} {{.CLI_ARGS}}
+      - gum confirm "Simulated changes above. Are you sure you want to continue?"
+      - ansible-playbook -vv launchpad-core/ansible/zfs_install.yaml -i {{.INVENTORY_PATH}} {{.CLI_ARGS}}
+
   apply-k0s:
     interactive: true # for interactive scripts
     desc: Joins all hosts into a K0s cluster as defined in the inventory
     cmds:
-      - ansible-playbook -vv launchpad-core/ansible/install_k0s_cluster.yaml --check --diff -i {{.INVENTORY_PATH}} {{.CLI_ARGS}}
-      - gum confirm "Simulated changes above. Are you sure you want to continue?"
+      # - ansible-playbook -vv launchpad-core/ansible/install_k0s_cluster.yaml --diff -i {{.INVENTORY_PATH}} {{.CLI_ARGS}}
```

Comment:
@chriswessels running the ansible-playbook with --check mode here doesn't make sense at all because most tasks would be skipped due to dependency on results from previous tasks. Should we remove ln 28 and 29?